pages/workflow/PKPWorkflowHandler.inc.php

```diff
@@ -238,7 +241,7 @@ function editorDecisionActions($args, $request) {
 		import('lib.pkp.classes.linkAction.request.AjaxModal');
 		$editorActions = array();
 		$lastRecommendation = $allRecommendations = null;
-		if (!empty($editorsStageAssignments) && (!$reviewRoundId || $reviewRoundId == $lastReviewRound->getId())) {
+		if (!empty($editorsStageAssignments) && (!$reviewRoundId || ($lastReviewRound && $reviewRoundId == $lastReviewRound->getId()))) {
```

Comment:
Merged: https://github.com/pkp/pkp-lib/commit/9ca9b04886b056f6a25fe921fa556dc97b3f6ecb

Reply:
Oops, that should be https://github.com/pkp/pkp-lib/commit/a744079ae8f8775b399efb9da9257e93bf0b6d4f